README.md

```diff
@@ -345,22 +343,18 @@ Install man pages when not building from git source.
 
 It's noteworthy that when not using `KERL_BUILD_DOCS=yes`, the docset that may be downloaded can be up to 120MB.
 
-
 ### KERL_INSTALL_HTMLDOCS
 
 Install HTML documentation when not building from git source.
 
 It's noteworthy that when not using `KERL_BUILD_DOCS=yes`, the docset that may be downloaded can be up to 120MB.
 
-
 ### KERL_SASL_STARTUP
 
 Build OTP to use SASL startup instead of minimal (default, when var is unset).
 
-
```

Comment:
I'm not sure if the next section is correct, though. It seems it's either a title with no children or sub-titles not marked as such (starting from after-the-first).

Reply:
Hm, it does seem strange to have a heading with no children - I wonder if we just plain forgot to document them or if they were somehow accidentally deleted...

Reply:
How should I proceed?

Reply:
Let's just leave it for another day